packages/cli/src/rpc/search/search.ts

```diff
@@ -15,13 +18,29 @@ export type HandlerOptions = {
 };
 
 export async function handler(
-  appmapDir: string,
-  appmaps: string[] | undefined,
   query: string,
+  appmaps: string[] | undefined,
   options: HandlerOptions
 ): Promise<SearchRpc.SearchResponse> {
+  const config = configuration();
+  const { directories } = config;
   let appmapSearchResponse: SearchResponse;
   if (appmaps) {
+    const results = appmaps
+      .map((appmap) => {
+        let directory: string | undefined;
+        if (directories.length === 1) directory = directories[0];
+        else directory = directories.find((dir) => appmap.startsWith(dir));
+        if (!directory) return undefined;
```

Comment:
This is a bit awkward. We have to choose a directory to report for the appmap. Make some guesses / defaults.